Before.Xamarin.Android.sln.targets

```diff
@@ -3,4 +3,5 @@
   <Import Project="$(MSBuildThisFileDirectory)build-tools\scripts\ImportExportDocs.targets" />
   <Import Project="$(MSBuildThisFileDirectory)build-tools\scripts\PrepareWindows.targets" Condition=" '$(OS)' == 'Windows_NT' " />
   <Import Project="$(MSBuildThisFileDirectory)build-tools\scripts\RunTests.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)tests\api-compatibility\api-compatibility.targets" />
```

Comment:
Is this a new file? Should this be included on your changes?

Reply:
Doh!  Added.